node-unix-interface.md

```diff
@@ -0,0 +1,11 @@
+# The Node-Unix interface
+
+Imagine being able to build scripts that can fit into any pipeline of Unix
+commands at any place. Not only is that possible with Node.js, it's easy! Node
+provides easy access to everything you need to create powerful system level
+utilities, and following the Unix philosophy of modularity and reusability
+enables developers to write incredibly powerful and flexible command line
+applications. This talk will introduce the concepts of Unix command pipelines,
+input and output stream handling in Node.js, and effective API design for Node
+scripts. By the end, developers will be able to write custom scripts that hook
```

Comment:
Hmm... I would say this would be try if it were a workshop, but with a simple conference talk will they really be able to? Or will they know how to _begin_ developing a custom script?